free-courses-en.md

```diff
@@ -5,6 +5,7 @@
 * [Angular](#angular)
 * [Assembly](#assembly)
 * [Bash / Shell](#bash--shell)
+* [BlockChain](#blockchain)
```

Comment:
"BlockChain" isn't a language. It's high time we created a ###language_agnostic####blockchain category, there are already two books on bitcoin that could be moved from ####misc
Wikipedia has "blockchain" as a single word. I don't see it in intercap style much.